Image/source/image.css

```diff
@@ -0,0 +1,25 @@
+.image-container{
+    width: 30rem;
+    
+
```

Comment:
you can remove the empty lines

Reply:
Updated,  kindly check it whenever you're free